erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -38,18 +39,22 @@ WebRtcConnection::WebRtcConnection(std::shared_ptr<Worker> worker, const std::st
   sinkfbSource_ = this;
   globalState_ = CONN_INITIAL;
 
-  fec_receiver_.reset(webrtc::UlpfecReceiver::Create(this));
-
-  slideshow_handler_.reset(new RtpVP8SlideShowHandler(this));
-  audio_mute_handler_.reset(new RtpAudioMuteHandler(this));
-  bwe_handler_.reset(new BandwidthEstimationHandler(this, worker_));
+  slideshow_handler_ = std::make_shared<RtpVP8SlideShowHandler>(this);
+  audio_mute_handler_ = std::make_shared<RtpAudioMuteHandler>(this);
+  bwe_handler_ = std::make_shared<BandwidthEstimationHandler>(this, worker_);
+  fec_handler_ = std::make_shared<FecReceiverHandler>(this);
+  rtcp_processor_handler_ = std::make_shared<RtcpProcessorHandler>(this, rtcp_processor_);
 
   // TODO(pedro): consider creating the pipeline on setRemoteSdp or createOffer
   pipeline_->addFront(PacketReader(this));
+  pipeline_->addFront(rtcp_processor_handler_);
+  pipeline_->addFront(IncomingStatsHandler(this));
+  pipeline_->addFront(fec_handler_);
```

Comment:
This is not specific to this PR but it's easier to see now with the handler stack here... Could the `FecReceiverHandler` affect the bitrate estimation in `BandwidthEstimationHandler`?